src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid var(--white);
+  border-radius: 5px;
+
+  &__image {
+    margin-left: 3px;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__title {
+    margin-top: 36px;
+    margin-bottom: 0;
+    font-size: 12px;
+    font-style: normal;
+    line-height: 18px;
+    font-weight: 500;
+    color: var(--main-accent);
+  }
+
+  &__product-id {
+    margin-top: 4px;
+    color: var(--secondary);
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+
+  }
+
+  &__rate {
+    display: flex;
+    height: 16px;
+    margin-top: 16px;
+    justify-content: space-between;
+  }
+
+  &__rate-review {
+    display: flex;
+    margin: 0;
+    flex-direction: column;
+    justify-content: flex-end;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    color: var(--main-accent);
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-name {
+    margin: 0;
+    color: var(--secondary);
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__price-count {
+    margin: 0;
+    color: var(--main-accent);
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__button {
+    display: block;
+    margin-top: 16px;
+    width: 166px;
+    height: 40px;
+    text-align: center;
+    text-decoration: none;
+    color: var(--white);
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 40px;
+    text-transform: uppercase;
+    background-color: var(--blue-accent);
+    border-radius: 5px;
+    border: 1px solid transparent;
+
+    &:hover {
+      box-sizing: border-box;
+      color: var(--blue-accent);
+      background-color: var(--white);
+      border-color: var(--blue-accent);
+    }
```

Comment:
add transition for this values in `&__button`

Reply:
✅